### ISSUE TITLE
Question about Loss Function

### ISSUE THREAD

**Issue by AwayahShahid** [MIC-DKFZ/MultiTalent#13]
Hi, I see that in the loss function you use, you state: 
"We modify the loss function to be calculated only for classes that were annotated in the corresponding partially labeled dataset "

Does this also mean that no loss is calculated for the background class "0"? As it could be true 0 or another class that's not labelled in the particular training instance.
Can you please point me to where this loss is implemented in the code?

**Comment by AwayahShahid** [comment]
I am able to find and study the compute_loss function that uses BCE with logits. 
What I understand from glancing through it is, that valid regions/targets are all labelled **non-background** regions.

**Comment by constantinulrich** [comment]
Hi, i am not 100% sure if i understood what you mean, but there is no class 0 (background) anymore, because its always a binary decision for each pixel if a foreground class or not.

**Comment by constantinulrich** [comment]
> "We modify the loss function to be calculated only for classes that were annotated in the corresponding partially labeled dataset "

that's means just, that we only calculate the binary loss for each class that is annotated in the corresponding image

**Comment by AwayahShahid** [comment]
Yes, thanks very much!
